src/panels/config/automation/manual-automation-editor.ts

```diff
@@ -101,6 +102,13 @@ export class HaManualAutomationEditor extends LitElement {
 
   @state() private _sidebarKey?: string;
 
+  @storage({
+    key: "automation-sidebar-width-percentage",
+    state: false,
+    subscribe: false,
+  })
+  private _sidebarWidth = 40;
```

Comment:
Why do we save this as a percentage and not as a fixed pixel width?

Reply:
The min and max can still be percentages, only the value selected by the user have to be pixels.

Reply:
I don't agree, for now the default size is 35%. If I set it to 50%-50%, I should keep this ratio while resizing right?